ui/js/modal/modalCreditIntro/view.jsx

```diff
@@ -1,40 +1,64 @@
 import React from "react";
 import { Modal } from "modal/modal";
-import { CreditAmount } from "component/common";
+import { CreditAmount, CurrencySymbol } from "component/common";
```

Comment:
Although not too significant, defining `LBC` as a global constant instead of a component would remove the overhead of instantiating the component every time it's used and then calling the `render()` method.

Reply:
used a component since I thought it likely it will be a richer element (e.g. title text at least?). This was perhaps premature and it should have been a constant, but I'm going to leave this as-is.